nx/blocks/exp/exp.css

```diff
@@ -109,13 +113,6 @@ nx-profile {
     justify-content: center;
   }
 
-  .nx-exp-back {
```

Comment:
I don't think we want to get rid of this.

View > Edit = Back

None > Edit = Back (cancel)

Reply:
Again, think we can do this in your view only mode changes PR.